loops.js

```diff
@@ -18,7 +19,13 @@
 // klaar!
  // ==========================================
 
-
+for (let i = 0; i < 5; i++) {
+    if (i < 4) {
+        console.log('loop...')
+    } else if (i === 4) {
+        console.log('klaar!')
+    }
```

Comment:
Code geeft juiste uitkomst. In de uitwerking kun je ook gebruik maken van:
`    if (i < 4) {
        console.log('loop...')
    } else if (i === 4) {
        console.log('klaar!')
    }`